embedded/multierr/multierr.go

```diff
@@ -15,12 +15,45 @@ limitations under the License.
 */
 package multierr
 
-import "fmt"
+import (
+	"errors"
+	"fmt"
+)
 
 type MultiErr struct {
-	Errors []error
+	errors []error
```

Comment:
I'm wondering if it would be good to add custom `As` and `Is` methods (https://blog.golang.org/go1.13-errors#TOC_4.) so that native `errors.Is` and `errors.As` work here.

Currently checking if the multierror was caused by a specific low-level error (e.g. IO write failure) would require casting to multierr and using the `Includes` error.

Reply:
I though about it too, but wasn't fully convinced. But I think it'll be better